### ISSUE TITLE
Textures look filtered even know settings are set to none.

### ISSUE THREAD

**Issue by theboy181** [gonetz/GLideN64#241]
Background textures look filtered and it causes alignment issues with most background art. A good example would be Mario Golf title screen. if you compare against AL's RDP you will see that the textures look off.

**Comment by theboy181** [comment]
Figured I would upload so people could see how off the backgrounds look.

![capture](https://cloud.githubusercontent.com/assets/9299743/6977639/9038b6a4-d976-11e4-85c6-713ce08d820e.PNG)

**Comment by theboy181** [comment]
@gonetz 

Any chance that you could make it so that textures are not compressed looking?

 The screenshots above look like BAD .jpg on GLideN64

**Comment by AmbientMalice** [comment]
This is possibly a mild form of the texture alignment bug that afflicts a fair few games, namely in splash screens. https://github.com/gonetz/GLideN64/issues/471

**Comment by theboy181** [comment]
That might be going on a bit too, but if you look at the grass int hat picture you will clearly see the artifacts that are usually made from jpeg compression.

**Comment by AmbientMalice** [comment]
The lines on the left hand picture appear to be uniformly horizontal. Compression artifacts tend to be square-shaped.

**Comment by theboy181** [comment]
![capture](https://cloud.githubusercontent.com/assets/9299743/8369672/25dd7b80-1b72-11e5-81ec-498a8fbfd7d3.PNG)


I agree.. but it looks horrible if you ask me..

**Comment by theboy181** [comment]
IF you observe MK64 only the background is affected.. The text is fine. Doesn't this happen on most HLE plugins too ?

**Comment by theboy181** [comment]
@gonetz 
This bug should be a top priority, it would fix 50 or more games

**Comment by AmbientMalice** [comment]
>IF you observe MK64 only the background is affected.. The text is fine. Doesn't this happen on most HLE plugins too?

Some minor texture tiling issues are to be expected, especially at higher resolutions. But such signficant lines are not normal, IMO.

**Comment by gonetz** [comment]
Will be fixed when #800 will go to master

**Comment by theboy181** [comment]
![capture](https://cloud.githubusercontent.com/assets/9299743/18419508/83a1853e-7811-11e6-8ac6-3138999ebff3.PNG)


Was this fixed? what do I need to change?

**Comment by theboy181** [comment]
Running @ 1X native resolution seems to work.. However you loose Hi Res. and Rendering texrects in native resolution, also works, however the in game backgrounds look horrible..  
![capture](https://cloud.githubusercontent.com/assets/9299743/18419570/baa6b7ba-7812-11e6-8fbf-a4fe88c6cb15.PNG)

**Comment by theboy181** [comment]
unsure if I should keep my fingers crossed for this one. @gonetz is it possible to make textures react more like a VI FIlter where the filters are applied after the textures are aligned, and not to each texture before they are pieced together? 

VI filtering would be a nice enhancement, and make things look really sharp. I would pay for a campaign that started to pull off more unique features like this.

**Comment by theboy181** [comment]
I must of tested with GLideN64's default settings. The background looks much better with defined settings in this game but when you are running at a higher resolution they loose their sharpness. :(

**Comment by gonetz** [comment]
> is it possible to make textures react more like a VI FIlter where the filters are applied after the textures are aligned, and not to each texture before they are pieced together?

No. Plugin does not know game designers intents. "Render 2D elements in N64 resolution" option designed to improve rendering of 2D elements. It is best I can do.

Of course, it is possible to disable texture filtering at all, and then try to smooth the picture with post-processing. I'm not sure, is it a good idea. You may try to take Glide64, enable force point-sampled, run Mario Golf in HD resolution, take screen shot and apply some filters to it in graphics editor. If you will get better result than current texture filtering, I will add corresponding post-filter.

**Comment by thecoreyburton** [comment]
> The background looks much better with defined settings in this game but when you are running at a higher resolution they loose their sharpness. :(
Regarding filtering at "Render 2D elements in N64 resolution" at higher resolutions (in relation to image sharpness), there was some related discussion on #1788 that may be related.

**Comment by oddMLan** [comment]
Rendering with native texrects doesnt seem to cause any issues in the bg anymore. And it fixes the original issue.
![image](https://user-images.githubusercontent.com/7278372/68203498-2d75bc00-ff83-11e9-88fc-5d87d8481a24.png)
I guess this was fixed with the S2DEX fixes?

Ini entry forcing native texrects must be added to gliden64.custom.ini, probably.

**Comment by gonetz** [comment]
native texrects option may slowdown emulation on low-end PC.
